ts/DataGrid/DataGrid.ts

```diff
@@ -923,7 +946,8 @@ class DataGrid {
         }
 
         const headerEl = makeDiv(className);
-        headerEl.style.height = this.options.cellHeight + 'px';
+        headerEl.style.minHeight = this.options.cellHeight + 'px';
+        headerEl.style.maxHeight = this.options.cellHeight * 2 + 'px';
```

Comment:
Please add the description in the `cellHeight` API that the header height might increase its size in certain cases.